tools/release-posts/templates/release.ejs

```diff
@@ -44,7 +32,7 @@
 <!-- wp:paragraph -->
 <p>
   These are just some of the changes that are included in WooCommerce <%= displayVersion %>. 
-  You can find the complete changelog for this release in the <a href="https://github.com/woocommerce/woocommerce/blob/release/<%= displayVersion %>/changelog.txt" target="_blank">changelog.txt file</a>.
+  You can find the complete changelog for this release in the <a href="https://github.com/woocommerce/woocommerce/blob/<%= currentBranch %>/plugins/woocommerce/readme.txt">readme.txt file</a>.
```

Comment:
Why did you remove target blank? Its quite convenient imo to have links open new tab especially when reading and jumping between documentation.

Reply:
@samueljseay generally my opinion is that the user should decide on their end how and where they'd like a link to open. E.g., I usually use Command-click if I want things to load in a background tab. I had assumed the basic philosophical stance behind it ("let the client decide this") to be fairly uncontroversial, so I did not comment on this initially. Happy to reconsider, but fundamentally I don't think we can and should be deciding which links all readers will want to open in a new window.

Reply:
@lsinger no probs, just a philosophical difference and I'm fine with that. I think the opinions seem to differ but for reference material when reading an article I think it's handy since you don't want to leave the reference material while reading. That said I see super mixed opinions on it, so happy to remove 😄